packages/frontend/src/pages/Tile/components/TableBanner/ImportCsvButton.tsx

```diff
@@ -67,14 +77,17 @@ const ImportCsvModal = ({ onClose }: { onClose: () => void }) => {
 
   const [importStatus, setImportStatus] = useState<
     'ready' | 'creating columns' | 'importing' | 'completed' | 'error'
```

Comment:
noob question: actually why isn't this put as a type like how you did for `TILE_CREATE_MODE` ah?